samples/Client/DaprClient/StateStoreETagsExample.cs

```diff
@@ -21,30 +21,63 @@ public override async Task RunAsync(CancellationToken cancellationToken)
         {
             var client = new DaprClientBuilder().Build();
 
-            // Save state which will create an etag for the state entry
-            await client.SaveStateAsync(storeName, stateKeyName, new Widget() { Size = "small", Color = "yellow", }, cancellationToken: cancellationToken);
+            // Save state with a null etag. This will throw an exception
+            try
+            {
+                await client.TrySaveStateAsync<Widget>(storeName, stateKeyName,  new Widget() { Size = "small", Color = "yellow", }, null, cancellationToken: cancellationToken);
+            }
+            catch(ArgumentException)
+            {
+                Console.WriteLine($"TrySaveStateAsync with null etag threw ArgumentException");
+            }
```

Comment:
I don't think this belongs in the sample. How does it help users to see code that will never succeed?

Reply:
earlier, I had some handling for empty etags as well and the sample was trying to demonstrate that null and empty etags are handled differently. but now that I have taken out the empty etag stuff from here, I think this can be removed too.